activerecord/CHANGELOG.md

```diff
@@ -1,8 +1,17 @@
-*   Fix `change_table` setting datetime precision for 6.1 Migrations
+*   Change `has_secure_token` default to `on: :initialize`
 
-    *Hartley McGuire*
+    Change the new default value from `on: :create` to `on: :initialize`
+
+    Can be controlled by the `config.active_record.generate_secure_token_on`
+    configuration:
 
-*   Fix change_column setting datetime precision for 6.1 Migrations
+    ```ruby
+    config.active_record.generate_secure_token_on = :create
+    ```
+
+    *Sean Doyle*
+
+*   Fix `change_table` setting datetime precision for 6.1 Migrations
```

Comment:
Looks like one of these got lost in the rebase

Reply:
Thank you! I misread them as a duplicate of one another.

Reply:
That's a good point, maybe I should combine them 🤔